website/docusaurus.config.js

```diff
@@ -0,0 +1,171 @@
+// @ts-check
+// `@type` JSDoc annotations allow editor autocompletion and type checking
+// (when paired with `@ts-check`).
+// There are various equivalent ways to declare your Docusaurus config.
+// See: https://docusaurus.io/docs/api/docusaurus-config
+
+import {themes as prismThemes} from 'prism-react-renderer';
+
+// This runs in Node.js - Don't use client-side code here (browser APIs, JSX...)
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'Copilot by Detox',
+  tagline: 'Simplify your mobile app testing with intuitive, natural language commands, making automation faster and more efficient.',
+  favicon: '/img/homepage/detox-logo.png',
+
+  // Set the production url of your site here
+  url: 'https://your-docusaurus-site.example.com',
```

Comment:
I believe this is better to be updated.

Reply:
How does 'https://copilotbydetox.com/' sound?

Reply:
Where do you deploy it to, currently?

Reply:
I don't know actually. Can I just choose to deploy to this address?

Reply:
@LironMShemen no, to deploy to a domain we need to buy it.. That's not for now.

Reply:
Let's start with a plain deployment to github pages